src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -23,6 +23,12 @@
 	height: 100%;
 }
 
+.terminal-find .monaco-findInput {
+	width: 140px;
+	max-width: 140px;
+	min-width: 140px;
+}
```

Comment:
The class `monaco-findInput` is knowledge the terminal shouldn't know about and could easily break with a refactor. Can the matches support move into `SimpleFindWidget` as an option (`showResultCount?`) and that would get styled as part of that component?

Reply:
then SimpleFindWidget will need access to terminal service so that it can get the active instance's xterm to get the result count.

Reply:
or will need terminal find widget to update the terminal to which the simple find widget should apply when that changes